src/Engine/Core/FusSceneConverter.cs

```diff
@@ -35,8 +37,46 @@ public static SceneContainer ConvertFrom(FusFile fus)
                 return new SceneContainer();
             }
 
+            // if id is given set fields in FusFile
+            if (!string.IsNullOrEmpty(id))
+            {
+                fus.Header.LoadFilename = Path.GetFileName(id);
+                fus.Header.LoadPath = Path.GetDirectoryName(id);
+            }
+
             var instance = new FusFileToSceneConvertV1();
             var payload = (FusScene)fus.Contents;
+
+            // if path is set, update path dependent payload
+            if (!string.IsNullOrEmpty(fus.Header.LoadPath))
+            {
+                for (int i = 0; i < payload.ComponentList.Count; i++)
+                {
+                    if (payload.ComponentList[i] is FusMaterial matcomp)
+                    {
+                        if (matcomp.HasAlbedo)
+                        {
+                            matcomp.Albedo.Texture = fus.Header.LoadPath + "/" + matcomp.Albedo.Texture;
```

Comment:
Use Path.Combine here to be platform independent.

Reply:
I mean ... yes of thats what I meant to do!